### ISSUE TITLE
How to test pretrained model?

### ISSUE THREAD

**Issue by ghost** [pragyak412/Improving-Voice-Separation-by-Incorporating-End-To-End-Speech-Recognition#2]
Hello @pragyak412
I tried to get script for voice separation using this repository.
But I think there's no such testing here.
Could you share simple test script for voice separation and also ASR?
Thanks.

**Comment by pragyak412** [comment]
Hello @luys8611 ,
Testing code for Oracle Model(with ASR features): https://github.com/pragyak412/Improving-Voice-Separation-by-Incorporating-End-To-End-Speech-Recognition/blob/master/Oracle/test_real.py

Kindly check and let me know if you have any issue!

**Comment by ghost** [comment]
Does Oracle model for voice separation?
I used main.py script with "test" command using ConvTasNet mdoel.

**Comment by pragyak412** [comment]
Yes, Oracle is for voice separation, you can use that model and code are in 'Improving-Voice-Separation-by-Incorporating-End-To-End-Speech-Recognition/Oracle/'.
But currently, data loader is for the AVSpeech dataset, Kindly give me some time I will make a generic testing code as well.

**Comment by ghost** [comment]
Yes,
I downloaded some data using AVSpeech's script (Youtube video files) and converted to 8kHz audio.
Okay, That'll be better if you share some testing code.

**Comment by pragyak412** [comment]
Okay!
Kindly give me a week, as I am occupied with some prior commitments.
